src/style.css

```diff
@@ -0,0 +1,3 @@
+body {

```

Comment:
You should use better reset. Minimal:

*, *::before, *::after {
  box-sizing: border-box;
}
* {
  margin: 0;
}

Good read: https://www.joshwcomeau.com/css/custom-css-reset/